examples/src/test/scala/docs/testkit/TestKitDocExample.scala

```diff
@@ -0,0 +1,36 @@
+package docs.testkit
+
+import docs.cassandra.CassandraProjectionDocExample
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.TestSuite
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+
+//#projection-testkit-import
+import akka.actor.testkit.typed.scaladsl.ActorTestKit
+import akka.projection.testkit.scaladsl.ProjectionTestKit
+
+//#projection-testkit-import
+
+class TestKitDocExample extends TestSuite with Matchers with BeforeAndAfterAll with ScalaFutures {
+
+  //#projection-testkit
+  val actorTestKit = ActorTestKit("testing-projection")
```

Comment:
better to show it with ScalaTestWithActorTestKit, since that is probably what they will use